point_pattern.py

```diff
@@ -93,11 +119,18 @@ def mean_center(points):
     y : float
         Mean y coordinate
     """
-    x = None
-    y = None
+    x = 0
+    y = 0
+    for coor in points:
+        x += coor[0]
+        y += coor[1]
 
     return x, y
 
+sub = [(1,2), (3,4)]
+print mean_center(sub)
```

Comment:
This is what is throwing the error, print need parentheses.  You are 99% of the way there on this one.  Just need to divide the sums by the number of points.